pydis_site/apps/content/resources/guides/pydis-guides/help-channel-guide.md

```diff
@@ -22,49 +22,53 @@ If your question fits into the domain of one of our topical help channels, and i
+![Channel available message](/static/images/content/help_channels/available_message.png)
 
-If you're unable to type into these channels, this means you're currently **on cooldown**. In order to prevent someone from claiming all the channels for themselves, **we only allow someone to claim a new help channel every 15 minutes**. However, if you close your help channel using the `!dormant` command, this cooldown is reset early.
+In order to claim one, simply start typing your question into one of these channels. Once your question is posted, you have claimed this channel it will be moved down to the **Occupied Help Channels** category.
 
-![Channel available message](/static/images/content/help_channels/available_message.png)
-*This message is always posted when a channel becomes available for use.*
+At this point you will have the **Help Cooldown** role which will remain on your profile until your help channel is closed (with the `!closed` command) or goes dormant due to inactivity.
 
-## Q: For how long is the channel mine?
+# Frequently Asked Questions
 
-The channel is yours until it has been inactive for **30 minutes**. When this happens, we move the channel down to the **Python Help: Dormant** category, and make the channel read-only. After a while, the channel will be rotated back into **Python Help: Available** for the next question. Please try to resist the urge to continue bumping the channel so that it never gets marked as inactive. If nobody is answering your question, you should try to reformulate the question to increase your chances of getting help.
+## Q: How long does my help channel stay active?
 
+The channel remains open for **30 minutes** after your last message, but if another user posts in your channel, the closing will be delayed by 10 minutes.
+
+*You'll see this message in your channel once it goes dormant:*
 ![Channel dormant message](/static/images/content/help_channels/dormant_message.png)
-*You'll see this message in your channel when the channel is marked as inactive.*
 
-## Q: I don't need my help channel anymore, my question was answered. What do I do?
+## Q: No one answered my question. How come?
 
-Once you have finished with your help channel you or a staff member can run `!dormant`. This will move the channel to the **Python Help: Dormant** category where it will sit until it is returned to circulation. You will only be able to run the command if you claimed the channel from the available category, you cannot close channels belonging to others.
+The server has users active all over the world and all hours of the day, but some days are less active than others. If no one answered your question, feel free to claim another help channel a little later.
+
+If you still feel like your question is being overlooked, read our guide on [asking good questions](../asking-good-questions) to incease your chances of getting a response.
+
+## Q: I don't need my help channel anymore, my question was answered. What do I do?
```

Comment:
This is a comma splice 
```suggestion
## Q: I don't need my help channel anymore; my question was answered. What do I do?
```

Reply:
How about just "My question was answered. What do I do?"

Reply:
Sure.